### ISSUE TITLE
Waffle chart do not work in R 4.0.3

### ISSUE THREAD

**Issue by corinabioinformatic** [jenineharris/foundation-biostat-2020#1]
Dear Jenine, 
I am following all the tutorial, and when I run the waffle plot in my R4.0.3 the plot does not appear, or it appear just the legend. 
The video is here: 
Graphs for a single categorical variable https://youtu.be/mXUa0pMWw_I
And is in the minute 40.48 of the video
![bugwaffle](https://user-images.githubusercontent.com/14955321/141658570-957c98bf-7fab-454e-896f-aa78020ef00d.PNG)
.

**Comment by jenineharris** [comment]
Typically the waffle is actually there, just too small to see. Did you try the Zoom button?

Other than this, waffle is problematic for a lot of people and it is hard for me to know where it might be going wrong for you without being able to look at your whole file and the objects in your R document. Often the issue is that the data are entered as a data frame rather than a table of data.

Jenine

**Comment by corinabioinformatic** [comment]
Yes, I zoomed in both waffles  but in one only the legend appeared and in the other nothing appeared. Anyway than you so much for your fast reply, and it is a really nice explained tutorial @jenineharris . 
As soon as I make the waffle work (once I finish all the chapters) I will post it here .
